src/shell/Shell.cpp

```diff
@@ -1024,7 +1062,7 @@ static void parseArguments(int argc, char* argv[], ArgParser& argParser)
     }
 }
 
-int main(int argc, char* argv[])
+int main(int argc, char* argv[], char* envp[])
```

Comment:
AFAIK this third parameter `envp` for environments is not a standard feature.
What about using `getenv` instead?

Reply:
It does not specify, so as I looked into it I found some common ones that I think would be worth getting, for example `SHELL`, `PATH`, `PWD`, etc...

From ccp reference I found a link to some common ones, though this is POSIX standard:
https://pubs.opengroup.org/onlinepubs/9699919799/basedefs/V1_chap08.html#tag_08